src/modules/user/user.controller.ts

```diff
@@ -56,4 +59,105 @@ export const getUserDetailById = async (req: Request, res: Response, next: NextF
     } catch (error) {
         next(error);
     }
-};
\ No newline at end of file
+};
+
+
+
+
+
+
```

Comment:
what are these blanks?
And your comment style??? Please check what I had write in comment before every function